src/main/java/com/example/ajaxonboardingproject/service/mapper/CinemaHallMapper.kt

```diff
@@ -5,20 +5,20 @@ import com.example.ajaxonboardingproject.dto.response.CinemaHallResponseDto
 import com.example.ajaxonboardingproject.model.CinemaHall
 import org.springframework.stereotype.Component
 
-@Component
-class CinemaHallMapper : RequestDtoMapper<CinemaHallRequestDto, CinemaHall>,
-        ResponseDtoMapper<CinemaHallResponseDto, CinemaHall> {
-    override fun mapToModel(dto : CinemaHallRequestDto) : CinemaHall {
-        return CinemaHall(
-                description = dto.description,
-                capacity = dto.capacity
-        )
-    }
+fun CinemaHallMapper.mapToModel(dto : CinemaHallRequestDto) : CinemaHall {
+    return CinemaHall(
+            description = dto.description,
+            capacity = dto.capacity
+    )
+}
+
+fun CinemaHallMapper.mapToDto(model: CinemaHall): CinemaHallResponseDto {
+    return CinemaHallResponseDto(
+            id = model.id,
+            capacity = model.capacity,
+            description = model.description
+    )
+}
 
-    override fun mapToDto(model: CinemaHall): CinemaHallResponseDto {
-        return model.id?.let { CinemaHallResponseDto(
-                id = it,
-                capacity = model.capacity,
-                description = model.description)}!!
-    }
-}
\ No newline at end of file
+@Component
+class CinemaHallMapper

```

Comment:
the previous version of the `CinemaHallMapper` was better.

Now it looks like you use extension function just for using extension function :)